### PR TITLE
Feature/redesign

### PR DESCRIPTION
**Pull request by maxceem**
Redesign of the sidebar with a new Assets & Library tab, messages in a Drawer and moved Project Plan to the Dashboard.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "webpack-merge": "^4.1.1"
   },
   "dependencies": {
-    "appirio-tech-react-components": "git+https://github.com/appirio-tech/react-components.git#feature/connectv2",
+    "appirio-tech-react-components": "git+https://github.com/appirio-tech/react-components.git#feature/connectv2-redesign",
```
Comment (vikasrohit) [inline package.json]: @maxceem do we need this change?

File: src/routes/notifications/constants/notifications.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ export const GOTO = {
   PROJECT_DASHBOARD: '/projects/{{projectId}}',
   PROJECT_SPECIFICATION: '/projects/{{projectId}}/scope',
   PROJECT_PLAN: '/projects/{{projectId}}/plan',
-  TOPIC: '/projects/{{projectId}}/#feed-{{topicId}}',
```
Comment (vikasrohit) [inline src/routes/notifications/constants/notifications.js]: @maxceem Just to confirm, are we redirecting the old such links to new `messages` URL?

File: package.json (hunk 1 above)
Comment (maxceem) [reply package.json]: Created a PR to the `react-components` for putting changes for the redesign into the standard branch https://github.com/appirio-tech/react-components/pull/324

Comment (maxceem) [reply package.json]: As soon, as PR would be merged, can revert this change.

Comment (vikasrohit) [reply package.json]: Merged the PR https://github.com/appirio-tech/react-components/pull/324

File: src/routes/notifications/constants/notifications.js (hunk 2 above)
Comment (vikasrohit) [reply src/routes/notifications/constants/notifications.js]: Perfect. Thanks. I knew you have take care of it. 👍

**Pull request merged**